client/package.json

```diff
@@ -3,9 +3,13 @@
   "version": "0.1.0",
   "private": true,
   "dependencies": {
+    "firebase": "^6.3.5",
     "react": "^16.9.0",
     "react-dom": "^16.9.0",
-    "react-scripts": "3.1.1"
+    "react-redux": "^7.1.0",
+    "react-router-dom": "^5.0.1",
+    "react-scripts": "3.1.1",
+    "recompose": "^0.30.0"
```

Comment:
What is the purpose of `recompose` exactly?

Reply:
Recompose, just makes sure that if my component is wrapped in multiple HOCs at export, one HOC is applied before the other.  For example, my signUpForm component the my withRouter HOC is applied before my fireBase HOC for that component. But if hooks can do something similiar, we may replace it.